README.md

```diff
@@ -1 +1,32 @@
-# mealMaster
\ No newline at end of file
+# MealMaster
+
+Web app to help with your nutritional needs.
+
+## Description
+
+The MealMaster web app aims to meet the nutritinal needs of someone living a healthy balanced lifestyle. 
```

Comment:
Do you think we should include the user story in the description? Had some time free so tried rewriting it - 

The MealMaster web application aims to help users live a healthy, balanced lifestyle by assisting them in meeting their nutritional needs.

- As a User,
- I want to be able to identify the calories in ingredients and search for recipes,
- So that I can plan meals in advance and stick to my macro requirements.

MealMaster allows the user to select a specific date on a calendar and then calculate and identify their nutritional goals for that day through an interactive quiz.
	
The user can then search for ingredients and recipes to prepare for the days and weeks ahead. MealMaster allows the user to record nutritional information, giving feedback on the remaining nutritional allowances for the day. 

Information is stored in the user's browser storage, allowing them to easily monitor progress on return to the application.